### PR TITLE
feat(lib): add getInitialValue() function

### LINKED ISSUE #15: Add getInitialValue() function to get  the initial form value already stored in forms-manager
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[x ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Cannot get the initial value of form. This was in private. 

## Expected behavior

We would like to be able to obtain the initial value of the form. This value has been saved by the plug-in for the isDirty function. 
It is not possible to retrieve this value, as it is in private property. There is a method called setInitialValue(). It would be useful to have a method: getInitialValue().

/**
   *
   * Get the initial value for a control
   *
   * @example
   *
   * manager.getInitialValue('login');
   *
   */
  getInitialValue(name: keyof FormsState): any {
    return this.initialValues$$.get(name);
  }

## What is the motivation / use case for changing the behavior?

In our case, we would need to retrieve the initial value of the form, in order to make calculations afterwards to calculate the differences and track all the changes that have been made. And, like, we would use this plug-in that would store the initial state. It would be a shame to have to store this state somewhere else, when it's already stored in form-manager, and we could just retrieve it here.  

Is it ok for you to make a pull request to add this feature to the plug-in?

### PR DESCRIPTION
**Pull request by manudss**
To be able to obtain the initial value of the form.

Closes #15

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/ngneat/forms-manager/blob/master/CONTRIBUTING.md#commit
- [x ] Tests for the changes have been added (for bug fixes / features)
- [x ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #15 

## What is the new behavior?

add a function to get the initial value of a form.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -353,6 +349,25 @@ export class SettingsComponent {
 }
 ```
 
+### Set and get Initial Value
```
Comment (NetanelBasal) [inline README.md:352]: I think we can make it short and clear. Just write the two functions without the whole explanation around it.

Comment (manudss) [reply README.md:352]: juste like this : ?

### `setInitialValue(name, value)` - Set Initial Value

```ts
formsManager.setInitialValue('form', initialValue);
```
### `setInitialValue(name)` - Get Initial Value

```ts
formsManager.getInitialValue('form');
```
If the initial value was not defined, the get function will return undefined.

Comment (NetanelBasal) [reply README.md]: Yes

File: README.md [hunk 2]
```diff
@@ -353,6 +349,18 @@ export class SettingsComponent {
 }
 ```
 
+### `setInitialValue(name, value)` - Set the Initial Value
+
+```ts
+formsManager.setInitialValue('form', initialValue);
+```
+
+### `getInitialValue(name)` - Get the Initial Value or undefined if not exist.
```
Comment (NetanelBasal) [inline README.md:358]:
```suggestion
### `getInitialValue(name)` - Get the initial value or `undefined` if not exist.
```

File: README.md [hunk 3]
```diff
@@ -353,6 +349,18 @@ export class SettingsComponent {
 }
 ```
 
+### `setInitialValue(name, value)` - Set the Initial Value
```
Comment (NetanelBasal) [inline README.md:352]:
```suggestion
### `setInitialValue(name, value)` - Set the initial form's value
```

**Pull request merged**